ui-tests/src/test/java/org/openmrs/reference/DeleteFormTest.java

```diff
@@ -15,6 +16,7 @@
 /**
  * Created by nata on 24.06.15.
  */
+@Ignore
```

Comment:
Ignored the test until investigated.

Reply:
> Ignored the test until investigated.

Any reason for this comment? I can see that specific class is already ignore.

Reply:
@kdaud that is an outdated comment like you can see